docs/version0.19.md

```diff
@@ -44,6 +45,10 @@ The latest builds of OpenJDK with OpenJ9 for Java 8 and 11 at the AdoptOpenJDK c
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### Option to print code cache usage on stderr at JVM shutdown
+
+A new command line option `-XX:+PrintCodeCache` allows you to print the code cache memory usage on stderr at JVM shutdown. see [-XX:[+|-]PrintCodeCache](xxprintcodecache.md).
```

Comment:
`see` should be `See`. Or you can remove `see ...` and change `-XX:+PrintCodeCache` to [`-XX:+PrintCodeCache`](xxprintcodecache.md)`